.pre-commit-config.yaml

```diff
@@ -1,14 +1,19 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
--   repo: https://github.com/Yelp/detect-secrets
+  - repo: https://github.com/Yelp/detect-secrets
     rev: v1.3.0
     hooks:
-    -   id: detect-secrets
-        args: ['--baseline', '.secrets.baseline']
--   repo: https://github.com/pre-commit/mirrors-eslint
-    rev: 'v8.24.0'  # Use the sha / tag you want to point at
+      - id: detect-secrets
+        args: ["--baseline", ".secrets.baseline"]
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: "v8.24.0" # Use the sha / tag you want to point at
     hooks:
-    -   id: eslint
-        files: \.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
-        types: [file]
\ No newline at end of file
+      - id: eslint
+        files: \.[jt]sx?$ # *.js, *.jsx, *.ts and *.tsx
```

Comment:
Is this plugin automatically checking directories, or is this configured to only test for files in the root folder?